OpenRA.Renderer.Sdl2/Sdl2GraphicsDevice.cs

```diff
@@ -330,6 +330,43 @@ public void SetLineWidth(float width)
 			ErrorHandler.CheckGlError();
 		}
 
+		public Bitmap TakeScreenshot()
+		{
+			var rect = new Rectangle(Point.Empty, size);
+			var bitmap = new Bitmap(rect.Width, rect.Height, System.Drawing.Imaging.PixelFormat.Format32bppArgb);
+			var data = bitmap.LockBits(rect,
+				System.Drawing.Imaging.ImageLockMode.WriteOnly, System.Drawing.Imaging.PixelFormat.Format32bppArgb);
+
+			try
```

Comment:
To make sure that the data is unlocked, regardless of whether something threw an exception or not. Thus a `catch` is simply not needed.